pkgs/development/libraries/wayland/scanner.nix

```diff
@@ -42,7 +42,6 @@ stdenv.mkDerivation {
 
   meta = with lib; {
     inherit (wayland.meta) homepage license maintainers;
-    mainProgram = "wayland-scanner";
```

Comment:
I think the `wayland-scanner` pkg should contain a `mainProgram` entry with the correct vaule, currently it is pointing to the `-dev` derivation for some reason.

Reply:
sounds like that should be fixed in upstream `nix` then.